packages/datadog-plugin-jest/src/index.js

```diff
@@ -63,6 +67,55 @@ class JestPlugin extends Plugin {
     this.testEnvironmentMetadata = getTestEnvironmentMetadata('jest', this.config)
     this.codeOwnersEntries = getCodeOwnersFileEntries()
 
+    // TODO: set a timeout after which the promise is rejected
+    const gitMetadataPromise = new Promise((resolve, reject) => {
+      this.addSub('ci:git-metadata-upload:finish', err => {
+        if (err) {
+          reject(err)
+        } else {
+          resolve()
+        }
+      })
+    })
+
+    this.addSub('ci:jest:test-suite:skippable', ({ onResponse, onError }) => {
+      if (!this.config.isAgentlessEnabled || !this.config.isIntelligentTestRunnerEnabled) {
+        onResponse([])
+        return
+      }
+      // we only request after git upload has happened
+      gitMetadataPromise.then(() => {
```

Comment:
it's important that we fetch the skippable suites _after_ git metadata has been uploaded, otherwise the algorithm does not work